doc/reference/tooling/tcm/tcm_audit_log.rst

```diff
@@ -0,0 +1,335 @@
+..  _tcm_audit_log:
+
+Audit log
+=========
+
+..  admonition:: Enterprise Edition
+    :class: fact
+
+    |tcm_full_name| is a part of the `Enterprise Edition <https://www.tarantool.io/compare/>`_.
+
+|tcm_full_name| provides the audit logging functionality for tracking user activity
+and security-related events, such as:
+
+*   Successful and failed login attempts.
+*   Access to clusters, their configurations, data model, and stored data.
+*   Changes in the access control system: users, roles, passwords, LDAP configurations.
+
+The complete list of |tcm| audit events is provided in :ref:`Event types <tcm_audit_log_event_types>`.
+
+.. note::
+
+    |tcm| audit log records only events that happen in |tcm| itself.
+    For information about Tarantool audit logging, see :ref:`Audit module <enterprise_audit_module>`.
+
+Audit logging is disabled in |tcm| by default. To start recording events, you need
+to :ref:`enable <tcm_audit_log_enable>` and :ref:`configure <tcm_audit_log_config>` it.
+
+Audit log stores event details in the JSON format. Each log entry contains the
+event type, description, time, impacted objects, and other information that
+may be used for incident investigation. The complete list of fields is provided in
+:ref:`Structure of audit log events <tcm_audit_log_structure>`.
+
+In addition to writing audit logs, |tcm| has a built-in interface for reading and
+searching them. For details, see :ref:`Viewing audit logs <tcm_audit_log_view>`.
```

Comment:
I'd rephrase this smth like this (without an introductory phrase): `|tcm| also has built-in interface for reading and
searching audit logs.`